### DISCUSSION TITLE
Grant claim blocks for votes.

### DISCUSSION THREAD

**Discussion by benrobson** [GriefPrevention/GriefPrevention#793] [Q&A] [answered]
Hey,
The main Spigot wiki page says that  you can "Grant claim blocks automatically for votes, donations, etc (console command provided, other plugins required)."
Is there a link to any plugins that allow this?

**Comment by RoboMWM** [comment, marked as answer]
Any plugin that can run console commands. Pretty sure you can find a vote plugin that runs a command, etc.
